src/components/footer/LanguageNavigation.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+
+import { Link, StackDivider } from '@chakra-ui/react';
+
+import Text from '../text';
+import Stack from '../stack';
+import Center from '../center';
+
+const FooterLanguageNavigation: FC = () => {
+  return (
+    <Center>
+      <Stack
+        marginTop="12px"
+        spacing="24px"
+        direction="row"
+        divider={<StackDivider borderColor="gray.700" />}
+      >
+        <Link href="/de" color="white">
```

Comment:
Are we redirected to the home page here?

Reply:
Yes. It's the most straightforward approach for now. We can only ever handle a subset of all the links in the footer if we want to redirect to the current page in a different language as the paths are localized. Wich comes to withe some overhead so I went with the most straightforward approach for now.